include/openenclave/attestation/plugin.h

```diff
@@ -100,7 +97,6 @@ struct _oe_attester
      */
     oe_result_t (*get_evidence)(
         oe_attester_t* context,
-        uint32_t flags,
```

Comment:
This only breaks plugin authors which are a smaller group of people. One of the header installation I did in this PR is to fix plugin development. Without that fix, plugin will not build. Since we've not heard of any complaint about that, it's probably safe to assume there are very few that are actively developing plugin.
Plugin APIs were publish in the last release which was a month ago. The longer we delay the change, the harder it will be to introduce it.
If the consensus suggests the same API_VERSION should be done, we can add that. I just don't want to make it more complicated than necessary.

Reply:
@CodeMonkeyLeet any thoughts on versioning the plugin API? I think we should treat these APIs the same we do our normal public API surface.

Reply:
This only breaks plugin authors which are a smaller group of people. One of the header installation I did in this PR is to fix plugin development. Without that fix, plugin will not build. Since we've not heard of any complaint about that, it's probably safe to assume there are very few that are actively developing plugin.
Plugin APIs were publish in the last release which was a month ago. The longer we delay the change, the harder it will be to introduce it.
If the consensus suggests the same API_VERSION should be done, we can add that. I just don't want to make it more complicated than necessary.

---
In reply to: [383454992](https://github.com/openenclave/openenclave/pull/2579#discussion_r383454992) [](ancestors = 383454992)

Reply:
These appear to be public (or experimental) APIs and should follow the same API guidelines for public or experimental APIs. (See discussion in PR #2163)

Reply:
'flags' were designed specifically to indicate local or remote attestation for SGX. I reviewed other attestation designs and see no reason to keep the 'flags' parameter.

---
In reply to: [383578425](https://github.com/openenclave/openenclave/pull/2579#discussion_r383578425) [](ancestors = 383578425)